src/Microsoft.Health.Dicom.SqlServer/Features/ExtendedQueryTag/ExtendedQueryTagDataRowsBuilder.cs

```diff
@@ -91,6 +105,18 @@ private static void AddDateTimeRow(DicomDataset instance, List<InsertDateTimeExt
             }
         }
 
+        private static void AddDateTimeWithUtcRow(DicomDataset instance, List<InsertDateTimeExtendedQueryTagTableTypeV2Row> dateTimeRows, QueryTag queryTag)
+        {
+            DateTime? dateVal = DataTimeReaders.TryGetValue(
+                             queryTag.VR,
+                             out Func<DicomDataset, DicomTag, DicomVR, DateTime?> reader) ? reader.Invoke(instance, queryTag.Tag, queryTag.VR) : null;
+
+            if (dateVal.HasValue)
+            {
+                dateTimeRows.Add(new InsertDateTimeExtendedQueryTagTableTypeV2Row(queryTag.ExtendedQueryTagStoreEntry.Key, dateVal.Value, null, (byte)queryTag.Level));
```

Comment:
As we discussed, whether or not tagvalue contains timezone offset, the `TagValueUtc` should be set to some value, but here is `null`

Reply:
This is actually just part of the changes, kind of a place holder. Once this PR gets checked in, my following PR will contain backend changes which will update this code.